src/inputs/TogglableColorPicker.tsx

```diff
@@ -1,11 +1,17 @@
 import { Fragment } from 'react'
 import fontColorContrast from 'font-color-contrast';
 import { ColorResult, SketchPicker } from 'react-color';
+import { Color } from '../types'
 import { useState } from "react";
 
+
+const forceColorType = (colorString : string) : Color => (
```

Comment:
I'm getting a typescript error here:
```
src/inputs/TogglableColorPicker.tsx|8 col 58 error| Type 'string' is not assignable to type 'Color'. 
```

I probably still need to figure out how to typecast in this? or something.

Reply:
I changed the definition for color to just be hex and I'm still getting "src/inputs/TogglableColorPicker.tsx|8 col 58 error| Type 'string' is not assignable to type 'Color'."